frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-hub.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { nav } from '../../../integration-tests-cypress/views/nav';
+
+describe('Interacting with OperatorHub', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('displays OperatorHub tile view with expected available Operators', () => {
+    cy.log('navigate to OperatorHub');
+    nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
+    cy.url().should('include', `/operatorhub/ns/${testName}`);
+    cy.log('more than one tile should be present');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+  });
+
+  it('displays OperatorHub tiles filtered by "Provider Type"', () => {
+    cy.log('enable the Red Hat filter');
+    cy.byTestID('providerType-red-hat').click();
+    cy.log('more than one tile should be present');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+    cy.log('track which tile is first');
+    const firstOperatorWithRHFilterApplied = cy
+      .get('.catalog-tile-pf')
+      .first()
+      .find('.catalog-tile-pf-title');
+    cy.log('diable the Red Hat filter');
+    cy.byTestID('providerType-red-hat').click();
+    cy.log('enable the Certified filter');
+    cy.byTestID('providerType-certified').click();
+    cy.log('more than one tile should be present');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+    cy.log('the first tile for Certified should not be the same as Red Hat');
+    cy.get('.catalog-tile-pf')
+      .first()
+      .find('.catalog-tile-pf-title')
+      .should('not.equal', firstOperatorWithRHFilterApplied);
+  });
+
+  it('filters Operators by name', () => {
+    const operatorName = 'Alcide kAudit Operator';
+    cy.byTestID('search-operatorhub').type(operatorName);
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+    cy.get('.catalog-tile-pf')
+      .first()
+      .find('.catalog-tile-pf-title')
+      .should('have.text', operatorName);
+    cy.byTestID('search-operatorhub').clear();
+  });
+
+  it('displays "Clear All Filters" link when text filter removes all Operators from display', () => {
+    cy.log('enter a search query that will return zero results');
+    cy.byTestID('search-operatorhub').type('NoOperatorsTest');
+    cy.get('.catalog-tile-pf').should('not.exist');
+    cy.byLegacyTestID('catalog-clear-filters').should('exist');
+  });
+
+  it('clears text filter when "Clear All Filters" link is clicked', () => {
+    cy.byLegacyTestID('catalog-clear-filters').click();
+    cy.byTestID('search-operatorhub').should('be.empty');
+    cy.get('.catalog-tile-pf')
+      .its('length')
+      .should('be.gt', 0);
+  });
+
+  it('filters Operators by category', () => {
+    cy.log('click the second category filter');
+    cy.get('.vertical-tabs-pf-tab > a')
+      .eq(2)
```

Comment:
This seems somewhat prone to changing/breaking.  Anyway to find the tab/a more deterministacally?